### DISCUSSION TITLE
Does temperature make sense in transcription?

### DISCUSSION THREAD

**Discussion by OleguerCanal** [openai/whisper#406] [Q&A] [answered]
I'm likely wrong but as I understand, the temperature parameter is very interesting in generative models (such as GPT) to "enhance creativity". However, I'm not sure how it helps in the case of speech-to-text, where one wants to get the exact words being pronounced. Isn't that why we get so many hallucinations in whisper?

**Comment by jianfch** [comment, marked as answer]
By default, the temperature is not used for the first decoding attempt. It is only when the decoded results do not meet the `compression_ratio_threshold` or `logprob_threshold` that it resorts to temperature fallback.
According to Table 7 in the paper, using temperature fallback on average does improve the performance of long-form transcription.
![image](https://user-images.githubusercontent.com/28970749/197610694-fe37d657-718b-4452-a5c0-eb6bbe28afdb.png)

**Comment by JonathanFly** [comment]
What about translation. Do higher temperatures or other parameter changes lead to less literal translations? Skimming the paper I don't see similar stats for the translation task.

**Reply by cifarra** [reply]
That'd be cool
